ql/src/java/org/apache/hadoop/hive/ql/parse/repl/dump/log/IncrementalDumpLogger.java

```diff
@@ -60,4 +60,9 @@ public void eventLog(String eventId, String eventType) {
   public void endLog(String lastReplId) {
     (new IncrementalDumpEnd(dbName, eventSeqNo, dumpDir, lastReplId)).log(LogTag.END);
   }
+
+  @Override
+  public void endLog(long totalCount) {
```

Comment:
Same here

Reply:
Atleast one endLog has to be present. If you provide a implementation in the base class, there maybe a derived class which doesn't override.